pkg/utils/build/quay.go

```diff
@@ -103,3 +104,35 @@ func DoesTagExistsInQuay(imageURL string) (bool, error) {
 	}
 	return false, nil
 }
+
+func IsImageRepoPublic(quayImageRepoName string) (bool, error) {
+	isPublic, err := quayClient.IsRepositoryPublic(quayOrg, quayImageRepoName)
+	if isPublic {
+		return true, nil
+	} else if !isPublic && err == nil {
+		return false, nil
+	} else {
+		return false, err
+	}
+}
+
+func IsQuayOrgSupportsPrivateRepo() (bool, error) {
```

Comment:
Could we maybe change the name here? like
`DoesQuayOrgSupportsPrivateRepo` or just `QuayOrgSupportsPrivateRepo`

Reply:
Tried to keep the  function naming consistent with other function names in the same pkg like `DoesImageRepoExistInQuay`, `DoesTagExistsInQuay` , `IsImageRepoPublic` etc.